cli/src/cli-cmd-parser.c

```diff
@@ -4448,22 +4447,24 @@ cli_snap_create_parse(dict_t *dict, const char **words, int wordcount)
             goto out;
         i++;
         /* point the index to next word.
-         * As description might be follwed by force option.
+         * As description might be follwed by force option which is deprecated.
          * Before that, check if wordcount limit is reached
          */
     }
 
     if (strcmp(words[i], "force") == 0) {
-        flags = GF_CLI_FLAG_OP_FORCE;
-
+        cli_out(
+            "Warning: \'force\' option is deprecated and "
+            "should not be used in the future while "
+            "creating snapshot");
```

Comment:
I would add more information here or a link where more information can be found (now `force` is a noop and snapshot is only allowed if all bricks are online).

I would also add a `TODO` to remember to completely remove the `force` keyword after a couple of releases.

Reply:
Added a Todo and some info regarding all bricks beings online for command to execute.